docs/_scripts/pretalx2wtd.py

```diff
@@ -1,55 +1,110 @@
 # Generate a sessions YAML from Pretalx talk selections.
 # To use:
 # - Get an API token from https://pretalx.com/orga/me
+# - Update the parameters at the end of this file, if needed (e.g. conference name)
+# - Do: export PRETALX_TOKEN=<your token>
 # - Go to directory docs/_scripts
-# - Download Pretalx data:
-#   curl -H "Authorization: Token $TOKEN" https://pretalx.com/api/events/$EVENT/talks/ -o talks.json
 # - Run this script
-
+import shutil
 import sys
 sys.path.insert(1, '../_ext')
 
+import os
 from collections import OrderedDict
 from utils import slugify
-import json
+import requests
 from ruamel import yaml
 from ruamel.yaml.representer import RoundTripRepresenter
 import markdown
 
+SPEAKER_IMAGE_PATH = '../_static/img/speakers/'
+
 yamldoc = []
 # Prevent OrderedDict from being presented as YAML OMAP - we just want a regular YAML dict.
 yaml.add_representer(OrderedDict, RoundTripRepresenter.represent_dict, representer=RoundTripRepresenter)
 
 
-def convert_to_yaml(year, series, slug, pretalx_results, yaml_filename):
-    with open(pretalx_results) as json_file:
-        talks = json.load(json_file)
-
-        for index, talk in enumerate(talks['results']):
-            slug = slugify(talk['title'] + '-' + talk['speakers'][0]['name'])
-            yamldoc.append(OrderedDict([
-                ('title', talk['title']),
-                ('slug', slug),
-                ('series', series),
-                ('series_slug', slug),
-                ('year', int(year)),
-                ('speakers', []),
-                ('abstract', markdown.markdown(talk['abstract'])),
-            ]))
-
-            for s in talk['speakers']:
-                yamldoc[index]['speakers'].append(OrderedDict([
-                    ('name', s['name']),
-                    ('slug', slugify(s['name'])),
-                    ('avatar', s['avatar']),
-                ]))
-
-        yaml_obj = yaml.YAML(typ='safe')
-        yaml_obj.Representer = RoundTripRepresenter
-        yaml_obj.default_flow_style = False
-        with open(yaml_filename, 'w+') as yaml_file:
-            yaml_obj.dump(yamldoc, yaml_file)
+def convert_to_yaml(year, series, yaml_output, pretalx_slug):
+    if not os.environ.get('PRETALX_TOKEN'):
+        print('Error: PRETALX_TOKEN not found in environment variables.')
+        return
+    http_headers = {'Authorization': 'Token ' + os.environ['PRETALX_TOKEN']}
+    submissions_url = f'https://pretalx.com/api/events/{pretalx_slug}/submissions/?state=confirmed'
```

Comment:
`{pretalx_slug}` needs to available here. Not sure if you use any of the other

Reply:
I am confused. It is available. The script runs fine?